src/main/resources/static/js/login.js

```diff
@@ -44,14 +44,15 @@ jQuery(document).ready(function () {
             error: function (XMLHttpRequest, textStatus, errorThrown) {
                 var statusCode = XMLHttpRequest.status;
 
+                // These error messages are not standardized.
                 if (statusCode == 401) {
-                    window.logMessages("Error Status code: 401 'Unauthorized or bad credentials'");
+                    logMessage("Error Status code: 401 'Unauthorized or bad credentials'");
```

Comment:
Does all of these error messages have to start with 'Error' ? Isn't it enough with saying what went wrong?

Reply:
Probably not, it's kind of implied already. I'll fix this.